### ISSUE TITLE
Show type material for taxa with basionyms with type material

### ISSUE THREAD

**Issue by dhobern** [CatalogueOfLife/checklistbank#1238]
Here is an example without visible type material:

https://www.checklistbank.org/dataset/2362/taxon/8247

![image](https://github.com/CatalogueOfLife/checklistbank/assets/26148454/4e5ef8d2-0410-4796-9246-015f7f1f1440)

This includes a reference to the basionym. Clicking on the basionym shows that this has type material.

![image](https://github.com/CatalogueOfLife/checklistbank/assets/26148454/4bc6eaba-b227-48bd-add1-aa6ec35763ce)

Under such circumstances, shouldn't the type material also show as the type material for the combination?

**Comment by mdoering** [comment]
I agree we should probably aggregate all material for homotypic names.
We might end up with duplicates that way if people declare types again for all combinations, but I reckon we can filter them out if they are truely identical.

We might also want to reconsider the ColDP model and allow a many-to-many relationship between types and names?
For sharing I do not really like intermediate many to many connecting tables. But at least all multiple values for the `TypeMaterial.nameID`?

![](https://github.com/CatalogueOfLife/coldp/blob/master/docs/schema.png)
